Updated cucumber libraries & aligned the sbt version with jenkins con…

jaymdavey opened the pull request:
…figuration.

TomJKing commented on project/build.properties:
```diff
@@ -1 +1 @@
-sbt.version = 1.3.7
\ No newline at end of file
+sbt.version = 1.4.1
```
Add single line blank line break to end of file.

jaymdavey replied:
I've added this.

TomJKing reviewed (changes requested):
Just one minor comment on the code.

In addition can you ensure you update the README instructions with changes related to the upgrade.

Specifically:

- At line 64: Currently reads: `Glue: "classpath:steps/"`, should be `Glue: "classpath:steps"`
- New instructions after line 70: `Program agruments: --plugin
org.jetbrains.plugins.cucumber.java.run.CucumberJvm4SMFormatter`

jaymdavey commented:
> Just one minor comment on the code.
> 
> In addition can you ensure you update the README instructions with changes related to the upgrade.
> 
> Specifically:
> 
> * At line 64: Currently reads: `Glue: "classpath:steps/"`, should be `Glue: "classpath:steps"`

With library updates, api signatures have changed & it is now sufficient to just specify the package name. So it now reads just "`Glue: "steps"`

> * New instructions after line 70: `Program agruments: --plugin 
> org.jetbrains.plugins.cucumber.java.run.CucumberJvm4SMFormatter`

I've added usage of `Program arguments: --plugin
 org.jetbrains.plugins.cucumber.java.run.CucumberJvm5SMFormatter as it is working fine. I've added a line to use CucumberJvm4SMFormatter in case of exceptions.
